src/main/java/org/fogbowcloud/saps/engine/core/catalog/exceptions/CatalogException.java

```diff
@@ -0,0 +1,10 @@
+package org.fogbowcloud.saps.engine.core.catalog.exceptions;
+
+public class CatalogException extends RuntimeException{
```

Comment:
@ThiagoWhispher, please, configure your IDE to automatically detect problems such `RuntimeException{`

Reply:
Done